MapboxGLAndroidSDKTestApp/build.gradle

```diff
@@ -18,8 +17,6 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
     }
 
-    nativeBuild(["example-custom-layer"])
```

Comment:
shall we ticket out the tail work of adding back custom layer?

Reply:
yes: https://github.com/mapbox/mapbox-gl-native-android/issues/452